gazebo_plugins/src/gazebo_ros_camera.cpp

```diff
@@ -435,15 +435,21 @@ void GazeboRosCamera::Load(gazebo::sensors::SensorPtr _sensor, sdf::ElementPtr _
     camera_info_msg.k[8] = 1.0;
 
     // rectification
-    camera_info_msg.r[0] = 1.0;
-    camera_info_msg.r[1] = 0.0;
-    camera_info_msg.r[2] = 0.0;
-    camera_info_msg.r[3] = 0.0;
-    camera_info_msg.r[4] = 1.0;
-    camera_info_msg.r[5] = 0.0;
-    camera_info_msg.r[6] = 0.0;
-    camera_info_msg.r[7] = 0.0;
-    camera_info_msg.r[8] = 1.0;
+    // Get string from SDF and parse into Matrix3d for populating message
+    std::string default_rmat("1.0 0.0 0.0 0.0 1.0 0.0 0.0 0.0 1.0");
+    std::istringstream stream(_sdf->Get<std::string>("rectification_matrix", default_rmat).first);
+
+    ignition::math::Matrix3d rmat;
+    stream >> rmat;
+    camera_info_msg.r[0] = rmat(0,0);
+    camera_info_msg.r[1] = rmat(0,1);
+    camera_info_msg.r[2] = rmat(0,2);
+    camera_info_msg.r[3] = rmat(1,0);
+    camera_info_msg.r[4] = rmat(1,1);
+    camera_info_msg.r[5] = rmat(1,2);
+    camera_info_msg.r[6] = rmat(2,0);
+    camera_info_msg.r[7] = rmat(2,1);
+    camera_info_msg.r[8] = rmat(2,2);
```

Comment:
The linter needs a space after the commas here : ``rmat(0, 0);``

Reply:
done